### PR TITLE
Add possibility to style element root.

### PR DESCRIPTION
**Pull request by matoushybl**

### CODE REVIEW

File: Sources/Tokenizer/Styles/PropertyHelper.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+public struct PropertyHelper {
+    
+    public static func deserializeSupportedProperties(properties: [PropertyDescription], in element: SWXMLHash.XMLElement) throws -> [Property] {
+        var result = [] as [Property]
+        for (attributeName, attribute) in element.allAttributes {
+            guard let propertyDescription = properties.first(where: { $0.matches(attributeName: attributeName) }) else {
+                continue
+            }
+            //            guard
```
Comment (MatyasKriz) [inline Sources/Tokenizer/Styles/PropertyHelper.swift:18]: Do you think we could get rid of this dead code, or is it still relevant?

File: Sources/Tokenizer/Styles/PropertyHelper.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            //                throw TokenizationError(message: "// Could not materialize property `\(propertyDescription)` from `\(attribute)`")
+            //                #endif
+            //            }
+            result.append(property)
+        }
+        return result
+    }
+
+    public static func deserializeToolingProperties(properties: [PropertyDescription], in element: SWXMLHash.XMLElement) throws -> [String: Property] {
+        var result = [:] as [String: Property]
```
Comment (MatyasKriz) [inline Sources/Tokenizer/Styles/PropertyHelper.swift:33]: What is the advantage of a dictionary over an ordinary array here?

**Review by MatyasKriz — APPROVED** [review]
Noice, keep going!

File: Sources/Tokenizer/Styles/PropertyHelper.swift (hunk 1 above)
Comment (matoushybl) [reply Sources/Tokenizer/Styles/PropertyHelper.swift:18]: I'll investigate on what does it do and whether it was moved to somewhere else, but I'll keep it here until the big refactoring.

**Pull request merged**